src/util/bpf_progs.cpp

```diff
@@ -0,0 +1,207 @@
+#include <string>
+#include <unistd.h>
+#include <unordered_set>
+#include <vector>
+
+#include "log.h"
```

Comment:
Please keep the header blocks tidy:

```
<external headers>

<internal headers>
```

This is mixing both together.

(This is just the most common internal convention and I cleaned up most files to be consistent. As long as it's consistent I think it's easier to read.)

Reply:
Ack.